### PR TITLE
Move notifier to context.

### PR DESCRIPTION
**Pull request by vmihailenco**

### CODE REVIEW

File: src/client.coffee [hunk 1]
```diff
@@ -107,7 +107,7 @@ class Client
 
   notify: (err) ->
     defContext = {
-      language: 'JavaScript',
+      language: 'JavaScript'
```
Comment (kyrylo) [inline src/client.coffee]: Missing comma

**Comment by kyrylo** [comment]
LGTM

**Comment by zefer** [comment]
LGTM

File: src/client.coffee (hunk 1 above)
Comment (vmihailenco) [inline src/client.coffee]: The convention is to drop commas from dictionary definition (at least we do this in ng).

**Comment by chaffeqa** [comment]
This actually breaks all `addFilter` calls since one of the checks is to see if `n.notifier != null`

[This line](https://github.com/airbrake/airbrake-js/blob/master/src/client.coffee#L140)

**Comment by kyrylo** [comment]
Thanks. Would you mind filing an issue?

**Comment by vmihailenco** [comment]
@chaffeqa thanks, fixed in https://github.com/airbrake/airbrake-js/pull/176

**Comment by chaffeqa** [comment]
thx @vmihailenco! sry i didnt have time to file an issue last night, ive been slaving away at trying to figure out how to make the reporter play nicely with cordova, since the server 400's on requests with backtraces with `file:blah/www...` in them ><

I have a hackey solution to that, just waiting for things to slow down so I can clean up the code and submit a pull request

thanks a ton you guys for all the work on airbrake reporting!

**Comment by vmihailenco** [comment]
Can you please elaborate? I don't quiete understand what kind of backtrace server (Airbrake server I suppose?) does not accept.

**Comment by stevecrozz** [comment]
I think some clarification is in order. According to the Airbrake documentation, the notifier belongs at the top level: https://airbrake.io/docs/#create-notice-v3

Why is it now moved to the context hash?

**Comment by zefer** [comment]
@stevecrozz thanks for pointing that out. We have updated the docs but the CDN is being incredibly stubborn and still serving the old version. We'll have that fixed soon.

We moved it to the context hash because we feel it is semantically better. The API is backwards compatible.

**Pull request merged**